api/spec/src/billing/invoices/product.tsp

```diff
@@ -0,0 +1,44 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi3";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.OpenAPI;
+
+namespace Invoices;
+
+/**
+ * UnitPriceItem represents a product or service that is sold to the customer based on a unit price.
+ */
+/* Omitted fields:
+ * - alt_prices: 	AltPrices defines a list of prices with their currencies that may be used as an alternative to the item's base price.
+ * - origin: Country code of where this item was from originally.
+ * - ext: 	Extension code map for any additional regime specific codes that may be required.
+ * - identities: List of additional codes, IDs, or SKUs which can be used to identify the item. They should be agreed upon between supplier and customer.
+ */
+@friendlyName("BillingUnitPriceItem")
+model UnitPriceItem {
+  @summary("Primary reference code that identifies this item.")
+  ref?: string;
+
+  @summary("Special key used to classify the item sometimes required by some regimes.")
+  key?: Key;
```

Comment:
key/name/description, should this extend resource?
https://github.com/openmeterio/openmeter/blob/main/api/spec/src/types.tsp#L75-L110